cmd/root.go

```diff
@@ -42,6 +42,8 @@ func newRootCmd() *cobra.Command {
 	// --root.project, this adds a convenience on top to allow --project to be used
 	rootCmd.PersistentFlags().StringVarP(&(config.GetConfig().Project), "project", "p", "", "Specifies the Flyte project.")
 	rootCmd.PersistentFlags().StringVarP(&(config.GetConfig().Domain), "domain", "d", "", "Specifies the Flyte project's domain.")
+	rootCmd.PersistentFlags().BoolVarP(&(config.GetConfig().UseAuth), "useAuth", "u", false, "Used when talking with an oauth enabled admin")
```

Comment:
should this not be part of the flyteadmin config?

Reply:
This i had added temporarily until the useAuth option was available to be used from the admin.
We need Haytham's bind variable change to use the command line option from admin namespace.

For now i am able to use this from config file and hence removing this temporary workaround
```
admin:
  # For GRPC endpoints you might want to use dns:///flyte.myexample.com
  endpoint: dns:///localhost:8089
  insecure: true
  useAuth: true
logger:
  show-source: true
  level: 5
```